github/provider.go

```diff
@@ -103,13 +91,17 @@ var descriptions map[string]string
 func init() {
 	descriptions = map[string]string{
 		"token": "The OAuth token used to connect to GitHub. " +
-			"If `anonymous` is false, `token` is required.",
+			"`anonymous` mode is enabled if `token` is not configured.",
 
-		"owner": "The GitHub owner name to manage.",
+		"base_url": "The GitHub Base API URL",
 
-		"organization": "(Deprecated) The GitHub organization name to manage.",
+		"insecure": "Enable `insecure` mode for testing purposes",
 
-		"base_url": "The GitHub Base API URL",
+		"owner": "The GitHub owner name to manage. " +
+			"Use this field instead of `organization` when managing individual accounts.",
+
+		"organization": "The GitHub organization name to manage. " +
+			"Use this field instead of `owner` when managing organization accounts.",
```

Comment:
maybe to avoid possible confusion, we can add a `ConflictsWith` attribute to these 2 fields in the schema since they each come with their respective use cases

Reply:
also, we should update `index.html.markdown` with this provider schema change for the no longer deprecated organization field and now optional token field. would be important to note as well that if both owner and organization are not set or set with `null` or empty string, the provider will default to the authenticated user

Reply:
>  add a `ConflictsWith` attribute

Added in a recent commit.